index.js

```diff
@@ -0,0 +1,147 @@
+    let firstDay = new Date(YEAR, MONTH, 1).getDay(); //firstDay부터 일자 출력하기 위함
+    let calendar = document.querySelector(".js-calendarBody");
+
+    let row = calendar.insertRow();
+    for(i = 0; i < firstDay; i++){
+        cell = row.insertCell();
+    } // 공백 만들기
+    for(i=1; i <= lastDate; i++){
+        if(firstDay != 7){
+            cell = row.insertCell();
+            cell.setAttribute('id', [i]);
+            cell.innerHTML = [i];
+            firstDay += 1;
+        } else {
+            row = calendar.insertRow();
+            cell = row.insertCell();
+            cell.setAttribute('id', [i]);
+            cell.innerHTML = [i];
+            firstDay = firstDay - 6;
+        }
+    }
+}
+
+function deleteCalendar() {
+    const calendar = document.querySelector(".js-calendarBody");
+    while(calendar.rows.length > 0) {
+        calendar.deleteRow(calendar.rows.length -1);
+    }
+}
+
+// 오늘 일자 색변경
+function paintToday() {
+    const id = document.getElementById([today.getDate()]);
+//    today.getMonth() == MONTH)
+    if(today.getMonth() == MONTH && today.getFullYear() == YEAR){
+      id.classList.add(PAINTING);
+    }
+}
+
+// 버튼 클릭시 월 이동
+function prevMonth() {
+    const months = new Array("JAN", "FEB", "MAR", "APR", "MAY", "JUN", "JUL", "AUG", "SEP", "OCT", "NOV", "DEC");
+    // 현재 1월이면 update
+    if(MONTH - 1 == -1) {
+      MONTH = 11;
+      YEAR -= 1
+    } else {
+      MONTH -= 1;
+    }
+    month.innerText = `${months[MONTH]} ${YEAR}`
+    // day -> 그달의 첫째날로 변경
+    const week = new Array('SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT');
+    day.innerText = week[new Date(YEAR, MONTH, 1).getDay()];
+    // date ->1일로 변경
+    if(MONTH == new Date().getMonth() && YEAR == new Date().getFullYear()) {
+        date.innerText = new Date().getDate();
+    } else {
+        date.innerText = new Date(YEAR, MONTH, 1).getDate();
+    }
+    // deleteCalendar함수 실행
+    deleteCalendar();
+    // makeCalendar함수 실행
+    makeCalendar();
+    paintToday();
+
+}
+function nextMonth() {
+    const months = new Array("JAN", "FEB", "MAR", "APR", "MAY", "JUN", "JUL", "AUG", "SEP", "OCT", "NOV", "DEC");
+    // 현재 1월이면 update
+    if(MONTH + 1 == 12) {
+      MONTH = 0;
+      YEAR += 1
+    } else {
+      MONTH += 1;
+    }
+    month.innerText = `${months[MONTH]} ${YEAR}`
+    // day -> 그달의 첫째날로 변경
+    const week = new Array('SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT');
+    day.innerText = week[new Date(YEAR, MONTH, 1).getDay()];
+    // date ->1일로 변경
+    if(MONTH == new Date().getMonth() && YEAR == new Date().getFullYear()) {
+        date.innerText = new Date().getDate();
+    } else {
+        date.innerText = new Date(YEAR, MONTH, 1).getDate();
+    }
+    // deleteCalendar함수 실행
+    deleteCalendar();
+    // makeCalendar함수 실행
+    makeCalendar();
+    paintToday();
+}
+
+const prevBtn = document.getElementsByClassName("prevBtn")[0]; // array를 retun하므로 [0]입력
+const nextBtn = document.getElementsByClassName("nextBtn")[0];
```

Comment:
`querySelector`로도 할 수 있습니다.

```suggestion
const prevBtn = document.querySelector(".prevBtn");
const nextBtn = document.querySelector(".nextBtn");
```